x/merkledb/db.go

```diff
@@ -34,9 +33,7 @@ import (
 const (
 
 	// TODO: name better
-	rebuildViewSizeFractionOfCacheSize   = 50
```

Comment:
This isn't as meaningful since the cache size is now based on byte size rather than count.